.github/workflows/python-app.yml

```diff
@@ -23,19 +23,47 @@ jobs:
       uses: actions/setup-python@v3
       with:
         python-version: "3.10"
+    - name: Cache dependencies
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pylint pytest
+        pip install pylint
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Lint with pylint
       run: |
-        # stop the build if there are Python syntax errors or undefined names
```

Comment:
Is there any specific reason you removed flake8 check?

Reply:
sorry for that mistake i just overlooked it when rewriting the jobs, I'm sorry that was unintentional